src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -464,7 +464,7 @@ bool Profile::HasIcon() const noexcept
 std::wstring_view Profile::GetIconPath() const noexcept
 {
     return HasIcon() ?
-           std::wstring_view{ _icon.value().c_str(), _icon.value().size() } :
+           std::wstring_view{ *_icon } :

```

Comment:
In this case, we prefer to have the size explicitly passed to the constructor to eliminate any chance of a missing null char and overflowing a buffer.

Reply:
https://en.cppreference.com/w/cpp/string/basic_string/operator_basic_string_view says `std::basic_string::operator basic_string_view()` already constructs a string view covering the entire (possibly null-containing) string.

Reply:
I believe @fghzxm is correct -- this should do the right thing.  There is implicit conversion from string to string_view as appropriate.

Reply:
We like to be extra safe :)